_hackbars/bislam.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: Hey
+author: Bislam S
+---
+
+Let's become a better developers so we can figure to stop our planet from polution
```

Comment:
<----Let's become a better developers so we can figure to stop our planet from polution--->